### ISSUE TITLE
Failing test: IgxGrid Master Detail #grid Keyboard Navigation

### ISSUE THREAD

**Issue by zdrawku** [IgniteUI/igniteui-angular#8076]
## Description  
The following test fails often 

```
Should navigate to the first data row using Ctrl + ArrowUp when all rows are expanded.
     IgxGrid Master-Detail #grid Keyboard Navigation 
     Error: Expected false to be truthy.
```

Build reference - https://github.com/IgniteUI/igniteui-angular/pull/8030/checks?check_run_id=1055785164#step:8:1767
